docs/reference/execution-providers/CoreML-ExecutionProvider.md

```diff
@@ -0,0 +1,67 @@
+---
+title: CoreML
+parent: Execution Providers
+grand_parent: Reference
+nav_order: 13
+---
+
+
+# CoreML Execution Provider
+{: .no_toc }
+
+[Core ML](https://developer.apple.com/machine-learning/core-ml/) is a machine learning framework introduced by Apple. It is designed to seamlessly take advantage of powerful hardware technology including CPU, GPU, and Neural Engine, in the most efficient way in order to maximize performance while minimizing memory and power consumption.
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Minimum requirements
+
+The CoreML EP requires iOS devices with iOS 13 or higher, or Mac computers with macOS 10.15 or higher. It is recommended to use Apple devices equipped with Apple Neural Engine to achieve optimal performance.
+
+## Build CoreML EP
+
+For build instructions for iOS devices, please see the [BUILD page](../../how-to/build.md#ios-coreml-execution-provider).
+
+## Using CoreML EP in C/C++
+
+```c++
+Ort::Env env = Ort::Env{ORT_LOGGING_LEVEL_ERROR, "Default"};
+Ort::SessionOptions sf;
```

Comment:
nit: `sf` is a strange variable name for session options, but not a big deal.

Reply:
I am not exactly show why, sf is used in all EP docs